nft-example/src/lib.rs

```diff
@@ -0,0 +1,496 @@
+#![no_std]
+#![feature(const_btree_new)]
+
+use codec::{Decode, Encode};
+use gstd::{debug, exec, msg, prelude::*, ActorId};
+use primitive_types::{H256, U256};
+use scale_info::TypeInfo;
+
+pub mod payloads;
+<<<<<<< HEAD
```

Comment:
git rebase/merge is not completed properly. @LouiseMedova please check this.

Reply:
Thanks, fixed it